packages/acs-ui-javascript-loaders/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@internal/acs-ui-javascript-loaders",
```

Comment:
Should this package be a separate release so as to not make the main react bundle larger?